src/OpenReader.m

```diff
@@ -1085,4 +1083,14 @@ - (void)createFolders:(NSMutableArray *)params
     }
 }
 
+
+/**
+ Get the current timestamp
+ 
+ @return current timestamp as a string
+ */
++ (NSString *)currentTimestamp {
```

Comment:
The document is a bit outdated. You can declare class properties too:
```objc
@property (readonly, class, nonatomic) NSString *currentTimestamp;
```

Then they are implemented by a class method as a getter. Dot notation offers better semantics and Apple has been using it everywhere for accessors. Whether they are backed by an ivar is an implementation detail.